etl/src/v2/workers/apply.rs

```diff
@@ -344,7 +346,7 @@ where
 
     async fn process_syncing_tables(&self, current_lsn: PgLsn) -> Result<bool, Self::Error> {
         let table_replication_states = self.active_table_replication_states().await?;
-        info!(
+        debug!(
```

Comment:
Made this debug because this method is called periodically and keeps printing the same line.